src/pages/index.tsx

```diff
@@ -1,49 +1,137 @@
+import { GetStaticProps } from "next";
 import Box from "@component/Box";
-import Footer from "@sections/landing/Footer";
+import { Footer2 } from "@component/footer";
+import Wrapper from "@sections/landing/Wrapper";
 import Section1 from "@sections/landing/Section1";
 import Section2 from "@sections/landing/Section2";
 import Section3 from "@sections/landing/Section3";
 import Section4 from "@sections/landing/Section4";
-import axiosInstance from "config/axiosInstance";
-// import Section5 from "@sections/landing/Section5";
+import Section6 from "@sections/landing/Section6";
+import Section9 from "@sections/landing/Section9";
+import SidenavBar from "@sections/landing/SidenavBar";
+import GroceryLayout from "@component/layout/GroceryLayout";
+import useScroll from "@hook/useScroll";
+import api from "@utils/__api__/landing";
+// data models
+import Service from "@models/service.model";
+import Product from "@models/product.model";
+import Category from "@models/category.model";
+import { CategoryItem } from "@models/categoryNavList.model";
+import { GroceryTwoCarouselItem } from "@models/carousel.model";
 
-type Props = {};
+// ========================================================
+type Grocery2Props = {
+  categories: Category[];
+  serviceList: Service[];
+  dairyProducts: Product[];
+  featuredProducts: Product[];
+  bestSellProducts: Product[];
+  bestHomeProducts: Product[];
+  navigationList: CategoryItem[];
+  mainCarouselData: GroceryTwoCarouselItem[];
+  testimonials: any[];
+  discountBanners: any[];
+};
+// ========================================================
+
+const GroceryTwo = (props: Grocery2Props) => {
```

Comment:
maybe we should leave generic name for index.tsx.

Reply:
yup, I forgot to change the name, that was grocery-2 file :D